shared/protocol/src/main/java/io/pravega/shared/protocol/netty/FailingReplyProcessor.java

```diff
@@ -204,4 +204,15 @@ public void tableEntriesRead(WireCommands.TableEntriesRead tableEntriesRead) {
     public void tableEntriesDeltaRead(WireCommands.TableEntriesDeltaRead tableEntriesDeltaRead) {
         throw new IllegalStateException("Unexpected operation: " + tableEntriesDeltaRead);
     }
+
+    @Override
+    public void errorMessage(WireCommands.ErrorMessage errorMessage) {
```

Comment:
All of the classes that extend FailingReplyProcessor should be modifed to properly handle the error.

Reply:
What constitutes 'properly handling the error?